students/sikorski_marcin/Lesson_2_-_Python_basics.py

```diff
@@ -0,0 +1,130 @@
+#Lesson_3
+#Leap Year
+year = int(input("Enter year to check: "))
+
+if (year % 4 == 0) and (year % 100 != 0) or (year % 400 == 0):
+    print('LEAP')
+else:
+    print('COMMON')
+
+#Chocolate bar
+first_value_of_portion_ortion = int(input("Enter first value: "))
+second_value_of_portion = int(input("Enter second value: "))
+number_of_squares = int(input("Enter number of squares: "))
+
+if (number_of_squares %first_value_of_portion_ortion == 0 and number_of_squares /first_value_of_portion_ortion < second_value_of_portion) or (number_of_squares %second_value_of_portion== 0 and number_of_squares /second_value_of_portion < first_value_of_portion_ortion):
```

Comment:
To jest zdecydowanie za długa linijka. Linia powinna być nie dłuższa niż 79 znaków, a już na pewno nie powinna przekraczać 119. W pycharmie mozna skonfigurować wyświetlanie marginesu.
Brakuje również kilku spacji między operatorami.
(Nie zauważyłem tego wcześniej) Warunek ten również można skrócić:
`number_of_squares /first_value_of_portion_ortion < second_value_of_portion`
i `number_of_squares /second_value_of_portion < first_value_of_portion_ortion`
jest to samo co:
`number_of_squares < first_value_of_portion_ortion * second_value_of_portion`